<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="note_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Info:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="tasks_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="list_view">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>841</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="file_search">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>530</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>File Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
